pyanaconda/modules/services/services.py

```diff
@@ -189,6 +201,30 @@ def set_setup_on_boot(self, value):
         self.setup_on_boot_changed.emit()
         log.debug("Setup on boot is set to %s.", value)
 
+    @property
+    def post_install_tools_disabled(self):
```

Comment:
Please, change it to `post_install_tools_enabled`. It is difficult to work with the negations.

Reply:
The original idea was to mirror the `post_install_tools_disabled` key in the user interaction config file, but this really looks weird as part of the DBus module API. 

I guess adding a note that the key will be written out only if `post_install_tools_enabled` == `False` should be able to achieve the same result.